src/runtime_src/core/include/xrt/xrt_hw_context.h

```diff
@@ -45,10 +45,13 @@ class hw_context : public detail::pimpl<hw_context_impl>
    *  - latency                // ??
    *  - frame_execution_time   // ??
    *  - priority               // ??
+   *  - enable_isp_channel     // toggle isp communication
+   *  - enable_acp_channel     // toggle acp communication
    *
    * Currently ignored for legacy platforms
    */
-  using qos_type = std::map<std::string, uint32_t>;
+  using cfg_param_type = std::map<std::string, uint32_t>;
+  using qos_type = cfg_param_type; //alias to old type
```

Comment:
it's better to replace qos_type with cfg_param_type everywhere to avoid confusion. if it is replaced everywhere, please remove this.

Reply:
This makes sense if only we could insist that existing clients change their code immediately.
I requested the alias in order not to break pipelines with existing tests that use qos_type.
Let's leave for now, rest of code is changed.

Reply:
sure soren.